packages/office-ui-fabric-react/src/components/Slider/Slider.scss

```diff
@@ -1,24 +1,143 @@
 @import '../../common/common';
 
-.root {
-  user-select: none;
-  margin-bottom: 8px;
-}
+.row {
```

Comment:
Also you're creating a ton of more css overhead here, when the difference between the two should be very minimal.